tracer/src/Datadog.Trace/AppSec/InstrumentationGateway.cs

```diff
@@ -62,7 +62,7 @@ private static void LogAddressIfDebugEnabled(IDictionary<string, object> args)
             {
                 foreach (var key in args.Keys)
                 {
-                    Log.Debug("Pushing address {Key} to the Instrumentation Gateway.", key);
+                    Log.Debug("DDAS-0009-00: Pushing address {Key} to the Instrumentation Gateway.", key);
```

Comment:
Seems to me it's DDAS-0008-00 ? 
![image](https://user-images.githubusercontent.com/8353486/145015810-f8abcf2d-9bd9-4a03-b74e-f39095a11ade.png)

Reply:
https://docs.google.com/spreadsheets/d/1vMp7QL5veRM4jwmW2XEyLZDnWMAbDf4i-YOooYYS_g4/edit#gid=1580748593

Reply:
And as I'm still going through the system tests, I see they also test for the actual` DDAS-009 Available addresses .* match needs for rules`... do we implement it in the end or is it a waf message at debug? 
still following conversation here: https://dd.slack.com/archives/C01U0KHJBB4/p1638955931290900 😅

Reply:
Fixed the 0009 to 0008 in 4a501544a. I think the real 0009 should be covered with the WAF.